apps/codelab/src/app/components/buttons-nav-bar/menu-fullscreen-widget/menu-fullscreen-widget.component.ts

```diff
@@ -0,0 +1,24 @@
+import { Component, OnInit, ElementRef } from '@angular/core';
+
+@Component({
+  selector: 'codelab-menu-fullscreen-widget',
+  templateUrl: './menu-fullscreen-widget.component.html',
+  styleUrls: ['./menu-fullscreen-widget.component.scss']
+})
+export class MenuFullscreenWidgetComponent implements OnInit {
+
+  constructor() { }
+
+  handleClick() {
```

Comment:
Call this smth like OpenFullScreen()
and do   document.documentElement.requestFullscreen(); directly here